tests/include-import/run.py

```diff
@@ -39,11 +39,12 @@ def main():
   cmd += " %s/%s.yang" % (this_dir, TESTNAME)
   os.system(cmd)
 
-  from bindings import include_import
+  from bindings import include_import  # NOQA: F401
```

Comment:
Could you comment above here as to what the reason for this `NOQA` is please?

Reply:
This one is for `module imported but unused`. I left the import there though so that it could be verified as importable. Did you want me to leave an inline comment for this?

As a side note, this will go away completely with the unittest refactor since the bindings generation is done 100% in memory.

Reply:
This sounds fine with me.